includes/preflight.inc

```diff
@@ -775,9 +776,9 @@ function _drush_find_commandfiles_drush() {
 
   // Also discover Drush's own annotation commands.
   $discovery = annotationcommand_adapter_get_discovery();
-  $annotation_commandfiles = $discovery->discover(DRUSH_BASE_PATH . '/lib/Drush', '\Drush');
+  $annotation_commandfiles = $discovery->discover(DRUSH_BASE_PATH . '/src', '\Drush');
```

Comment:
Did I forget to do this when I relocated everything to /src?  If so, why didn't everything break? Odd.